router/core/flushwriter.go

```diff
@@ -3,15 +3,24 @@ package core
 import (
 	"bytes"
 	"context"
+	"github.com/wundergraph/astjson"
+	"mime"
 	"net/http"
+	"time"
 
 	"github.com/wundergraph/graphql-go-tools/v2/pkg/engine/resolve"
 )
 
 const (
-	WgPrefix             = "wg_"
-	WgSseParam           = WgPrefix + "sse"
-	WgSubscribeOnceParam = WgPrefix + "subscribe_once"
+	WgPrefix                   = "wg_"
+	WgSseParam                 = WgPrefix + "sse"
+	WgSubscribeOnceParam       = WgPrefix + "subscribe_once"
+	multipartBoundary          = "graphql"
+	multipartHeartbeatInterval = 5 * time.Second
+	multipartAcceptHeader      = "multipart/mixed;subscriptionSpec=\"1.0\", application/json"
```

Comment:
You can do a equal match here. Client could use spaces and it would be still valid. You have to parse the header with https://pkg.go.dev/mime